src/main/java/bio/terra/service/dataset/flight/delete/DatasetDeleteFlight.java

```diff
@@ -74,6 +74,7 @@ public DatasetDeleteFlight(FlightMap inputParameters, Object applicationContext)
     }
     if (platform.isGcp()) {
       // TODO: Do this check for Azure datasets
+      addStep(new DeleteDatasetStoreProjectIdStep(datasetId, datasetService));
```

Comment:
Just checking: should this happen before or after then next step (validation)?

Reply:
This just stores the project resource id, it should be fine to do it here. But, can we move that comment on the line before this step to stay before the DeleteDatasetGcpValidateStep? 
```suggestion
      addStep(new DeleteDatasetStoreProjectIdStep(datasetId, datasetService));
      // TODO: Do this check for Azure datasets
```